### PR TITLE
docs: introduction section adjustment

### PR DESCRIPTION
**Pull request by adela-bytebase**
Add a new section "introduction" instead of one file "what is bytebase", which affects the default link pointing to documentation. Moreover, it needs @boojack to help expand this section by default

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```
Comment (boojack) [inline pnpm-lock.yaml:1]: `pnpm-lock.yaml` shouldn't be changed

File: content/docs/en/_layout.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # docs_layout
 
-## [👀 What is Bytebase](/what-is-bytebase)
+## 👀 Introduction
+### [What is Bytebase](/introduction/what-is-bytebase)
```
Comment (boojack) [inline content/docs/en/_layout.md:4]:
```suggestion

### [What is Bytebase](/introduction/what-is-bytebase)

```

**Review by boojack — CHANGES_REQUESTED** [review]

File: content/docs/en/introduction/what-is-bytebase.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Bytebase complements the existing cloud provider's database platforms or the company's internal database operation platforms. While those platforms take care of the database instance level operations (e.g. provisioning a database instance), Bytebase helps teams to use the provisioned database to build their application.
+
+## Key Features
+
+### Schema (DDL) and Data (DML) Change Review Workflow
+Like code review, Bytebase streamlines the database change process. Within a single workflow, a database change can be reviewed and deployed from the dev environment all the way to the production environment.
+
+### SQL Quality Check
+Bytebase analyzes SQL changes to enforce rules in compliance with your organization's policy. The enforcement includes naming conventions, anti-SQL pattern detection and etc. Prod and non-prod environments can also enforce different rules respectively.
+### SQL Editor
```
Comment (tianzhou) [inline content/docs/en/introduction/what-is-bytebase.md:18]: nit: leave an empty line above

File: content/docs/en/introduction/what-is-bytebase.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+A web-based SQL Editor to query and export data. DBAs no longer need to give away sensitive database credentials when Developers need to access the data.
+
+### Version Control with VCS Integration
+Bytebase keeps the complete schema change history. It also integrates with VCS systems (e.g. GitLab). Teams can manage the SQL migration scripts in the VCS and trigger schema deployment on code commit.
+### Backup and Restore
```
Comment (tianzhou) [inline content/docs/en/introduction/what-is-bytebase.md:23]: nit: leave an empty line above

File: pages/docs/index.vue [hunk 5]
```diff
@@ -12,7 +12,7 @@ export default defineComponent({
   setup() {
     onMounted(() => {
       // `/docs` is an empty page as we need to redirect to `/docs/what-is-bytebase`.
```
Comment (tianzhou) [inline pages/docs/index.vue:14]:
```suggestion
      // `/docs` is an empty page as we need to redirect to `/docs/introduction/what-is-bytebase`.
```

**Review by tianzhou — DISMISSED** [review]
LGTM

File: content/docs/en/introduction/what-is-bytebase.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+title: What is Bytebase
+description: Bytebase is a database schema change and version control management tool for teams. It consists of a web console and a backend. The backend has a migration core to manage database schema changes. It also integrates with VCS to enable version controlled schema management.
+---
```
Comment (tianzhou) [inline content/docs/en/introduction/what-is-bytebase.md:4]:
```suggestion
---

```

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**